rasterio/_warp.pyx

```diff
@@ -723,9 +769,11 @@ cdef class WarpedVRTReaderBase(DatasetReaderBase):
                  dst_width=None, width=None, dst_height=None, height=None,
                  src_transform=None, dst_transform=None, transform=None,
                  init_dest_nodata=True, src_alpha=0, add_alpha=False,
-                 warp_mem_limit=0, dtype=None, **warp_extras):
+                 warp_mem_limit=0, dtype=None, transformer_options=None, **warp_extras):
```

Comment:
@snowman2 I'd like to see the transformer options folded into warp_extras as we're doing with _reproject: https://github.com/mapbox/rasterio/blob/master/rasterio/_warp.pyx#L479.

Reply:
That definitely simplified things and brings consistency with WarpedVRT and _reproject. The only downside is that you are not warned when the options are ignored with earlier versions of GDAL.